segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/IndexAppend.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.segmentstore.server.host.handler;
+
+import io.pravega.shared.protocol.netty.ByteBufWrapper;
+import lombok.Data;
+
+/**
+ * Java class to manage index segment appends.
+ */
+@Data
+public class IndexAppend {
```

Comment:
This class appears to only be used by IndexAppendProcessor. If so let's make it a static inner class.

Also, I am not clear on the need for both IndexAppend and IndexEntry. It appears that IndexEntry is serialized into the bufferWrapper field. However this is only used to pass it from one function to the next within the same class. At this point the segment name is not used (so it could reasonably made into a distinct parameter, and this class could be just removed, and IndexEntry used)

Reply:
Removed this newly added class.